src/components/FilterGroup.tsx

```diff
@@ -69,6 +72,13 @@ export function FilterGroup({
     };
   }, [customCssClasses]);
 
+  const limited = filterOptions.length > showMoreLimit;
+  const [showAll, setShowAll] = useState<boolean>(!limited);
+
+  function toggleShowAll() {
```

Comment:
At first I thought we might want to wrap this in a `useCallback`, if only for consistency, but after thinking about it more we can probably omit it.

Out of curiosity, if you use an arrow function instead like `onClick={e => setShowAll(!showAll)}` does a react-perf eslint rule complain? I was thinking that it might be better to use an arrow function and an eslint-ignore comment, just to be explicit here. 

Since toggleShowAll is just passed into a regular `button` and not another component I don't think there's benefits of wrapping this in useCallback